docs/general/configuration_options.md

```diff
@@ -5,6 +5,7 @@ The following configuration options can be set on `ActiveModel::Serializer.confi
 ## General
 
 - `adapter`: The [adapter](adapters.md) to use. Possible values: `:attributes, :json, :json_api`. Default: `:attributes`.
+- `automatic_lookup`: Whether serializer should be automatically looked up or manually provided. Default: `true`
```

Comment:
actually, you can pass `adapter: false` in your render (or `options[:adapter] = false` if you're outside a controller) and it won't look up a serializer or use an adapter.

ref: 

- https://github.com/rails-api/active_model_serializers/blob/f3403c302c46fd470c2c6073d6cc6c364d9c5bd0/lib/active_model/serializable_resource.rb#L54-L58
- https://github.com/rails-api/active_model_serializers/blob/f3403c302c46fd470c2c6073d6cc6c364d9c5bd0/lib/action_controller/serialization.rb#L23-L40

That should work for you, and would still be useful to include in this PR, to call that out :)

It should also be mentioned in https://github.com/rails-api/active_model_serializers/blob/master/docs/ARCHITECTURE.md, I think